src/CollAction/Controllers/ProjectsController.cs

```diff
@@ -110,23 +129,16 @@ public ViewResult StartInfo()
                 }
                 catch (DbUpdateException ex)
                 {
-                    if (ex.InnerException is PostgresException)
```

Comment:
* Validation this way is probably most efficient, but I really would rather put validation like this either in the viewmodel or just as a separate query at the start (AnyAsync(project => project.Name.Equals..)). I realize that might lead to rare race conditions (two people submitting projects with the same same at the exact same time), but it's more important that validation (exception code checking isn't really self-explanatory) is clear rather than a small gain in efficiency in a workflow that'd only be used when creating projects. Worst case scenario, one in a million people get a generic, error creating project, and retry. But our validation is a lot more clear.
* In the future you could use exception filters for stuff where the catch(ExceptionType.. isn't sufficient